### ISSUE TITLE
ctags-exuberant: Unknown option: --extras

### ISSUE THREAD

**Issue by GertBurger** [majutsushi/tagbar#556]
It looks like a recent change broke compatibility with exuberant-ctags. 

Error in vim messages is:

`ctags-exuberant: Unknown option: --extras`

Docs says version 5.5 is required and Ubuntu ships with 5.9

**Comment by alerque** [comment]
This is a duplicate of #552 which I fixed yesterday in 4b51aa4. I'm a little surprised to see a new report of it. @GertBurger can you confirm you are actually using the latest `master` and not a version from 2 days ago when I broke this?

By the way the docs should be fixed because 5.5 is just a minimum, not a maximum version for `ctags`. I'll leave this issue open until that is clarified. Note that `universal-ctags` is **very** highly recommended over any version of the original and very dated `ctags` though.

**Comment by GertBurger** [comment]
You are right, I updated about 30 hours ago and never bother to update again. Sorry for the time waster.

I'll see if my other apps are compatible with `universal-ctags` and then decide.

**Comment by alerque** [comment]
No worries @GertBurger, I was the one that let that breakage slip through. I'm glad to hear it works for you now.

I think on Ubuntu you can install `universal-ctags` in parallel to the original, but I'm not actually a user. On Arch and many other distros they have just replaced the `ctags` with the fork entirely, so I would be surprised to learn that it would break anything for you.

**Comment by jdhao** [comment]
met this issue also on macOS. What works for me is to uninstall the old ctags and install universal-ctags instead:

```
brew remove ctags
brew install universal-ctags
```
